test/mocha-global.js

```diff
@@ -1,10 +1,23 @@
-import { after } from 'mocha'
+import { before, after } from 'mocha'
 import { cleanupTestSpaces } from './helpers'
 
-after('clean up test spaces', async () => {
+const housekeeping = async () => {
   try {
     await cleanupTestSpaces()
-  } catch {
-    // ignore if the cleanup fails. Usually fails locally due to missing credentials
+  } catch (err) {
+    if (err.message === 'Missing credential') {
+      console.log('Skipped deletion of old test spaces due to missing credentials.')
+    } else {
+      console.log('Skipped deletion of old test spaces. Error:', err.message)
+    }
```

Comment:
nitpick: you can probably get away with just 1 console.log tbh, you can let other people check out the error etc

Reply:
This is intentional, as we expect the credential error to happen most of the time, and as it can basically be ignored, we don't want to log the word "Error" in that case (see related discussion [here](https://github.com/contentful/contentful-management.js/pull/1480#discussion_r968598636)).

Reply:
hmm gotcha 👍🏽